install/kubernetes/cilium/README.md

```diff
@@ -81,7 +81,7 @@ contributors across the globe, there is almost always someone available to help.
 | bpf.mapDynamicSizeRatio | float64 | `0.0025` | Configure auto-sizing for all BPF maps based on available memory. ref: https://docs.cilium.io/en/stable/concepts/ebpf/maps/#ebpf-maps |
 | bpf.masquerade | bool | `false` | Enable native IP masquerade support in eBPF |
 | bpf.monitorAggregation | string | `"medium"` | Configure the level of aggregation for monitor notifications. Valid options are none, low, medium, maximum. |
-| bpf.monitorFlags | string | `"all"` | Configure which TCP flags trigger notifications when seen for the first time in a connection. |
+| bpf.monitorFlags | string | `"syn fin rst"` | Configure which TCP flags trigger notifications when changed. |
```

Comment:
Reading the code, we do [keep track of seen flags for each direction](https://github.com/cilium/cilium/blob/26d31f00dd2858401d35a5afa0626c810ecbe207/bpf/lib/conntrack.h#L62) so it seems the intent is indeed to notify only when seeing a flag for the first time. If you noticed something different, it sounds like a bug.

Reply:
You are right, the code matches the documentation, but not behavior we observed. I will file a bug and link here as possible mitigation. Given the impact it has it may be better to include it until the underlying issue is solved

Reply:
Interesting... I tried both `none` and `ack syn rst fin` and they both show a flow every 5 seconds (for an ongoing connection). However, setting it to `all` has the effect as described in "Before".

I should try to reproduce it on `master` as I am currently running 1.11.7 with some modifications. I will have more info on Monday.

Feel free to close this PR and/or move it to bug. I agree that mitigation is not the best approach as it seems that not that many users complain about the current behavior.

Reply:
Hmm, that's very interesting. It's quite possible that there has been a regression for a while and no-one complained yet.

For what it's worth, the best documentation I'm aware of for the monitor aggregation today is at either https://docs.cilium.io/en/stable/concepts/kubernetes/configuration/#configmap-options or in the original PRs that introduced the features.